### PR TITLE
enhancement

### LINKED ISSUE #449: Enhancement: Make ARCHITECTURE.md easier to read
The sections for "Regenerate Jobs" and "Pipeline Jobs" have run-on sentences that tie my brain in knots. 

e.g. "These are the initial scripts that are the top level jobs that are invoked, etc etc"

I propose we break the sentences down into smaller sentences which are clearer.

e.g. "These are the initial scripts that are the top level jobs that are invoked," could be "These are the first scripts run by the top level jobs,", or similar. 

As a rule, this task is only complete once each sentence has two commas or less, and minimal redundancy.

### PR DESCRIPTION
**Pull request by lumuchris256**
enhnacement to the  ARCHITECTURE.md file 
fixes #449 
signed off by ; @lumuchris256 
cc  @adamfarley

### CODE REVIEW

File: docs/ARCHITECTURE.md [hunk 1]
```diff
@@ -16,17 +16,12 @@ A rough structure of the build is as follows
 
 #### Regenerate Jobs
 
-Jenkins CI jobs are generated by the [regeneration pipelines](https://ci.adoptopenjdk.net/job/build-scripts/job/utils/) in Jenkins. The documentation for this process can be found at [regeneration docs](https://github.com/adoptium/temurin-build/tree/master/pipelines/build/regeneration), however they generate the jobs
-at <https://ci.adoptopenjdk.net/job/build-scripts/> which are the builds that coordinate the entire process of nightly
-and release builds.
-
+Jenkins CI jobs are generated by the Jenkins [regeneration pipelines](https://ci.adoptopenjdk.net/job/build-scripts/job/utils/) , generated at <https://ci.adoptopenjdk.net/job/build-scripts/> and their builds  coordinate the entire process of nightly and release builds.
```
Comment (karianna) [inline docs/ARCHITECTURE.md:19]:
```suggestion
Jenkins CI jobs are generated by the Jenkins [regeneration pipelines](https://ci.adoptopenjdk.net/job/build-scripts/job/utils/). generated at <https://ci.adoptopenjdk.net/job/build-scripts/> and their builds coordinate the entire process of nightly and release builds.
```

File: docs/ARCHITECTURE.md [hunk 2]
```diff
@@ -16,17 +16,12 @@ A rough structure of the build is as follows
...
+Documentation for this process can be found at [regeneration docs](https://github.com/adoptium/temurin-build/tree/master/pipelines/build/regeneration)
 #### Pipeline Jobs
 
-These are the initial scripts that are the top level jobs that are invoked, this in turn invokes the downstream build
-jobs for each configuration/platform then coordinates deploying the results to GitHub. The majority of the code that
-defines this stage is contained within [build_base_file.groovy](https://github.com/adoptium/temurin-build/blob/master/pipelines/build/common/build_base_file.groovy).
+These are the first scripts run by the top level jobs, which in turn invoke the downstream build jobs for each configuration/platform, deploying the results to GitHub. Majority of the code that defines this stage is contained within [build_base_file.groovy](https://github.com/adoptium/temurin-build/blob/master/pipelines/build/common/build_base_file.groovy).
```
Comment (karianna) [inline docs/ARCHITECTURE.md:23]:
```suggestion
These are the first scripts run by the top level jobs, which in turn invoke the downstream build jobs for each configuration/platform, deploying the results to GitHub. The majority of the code that defines this stage is contained within [build_base_file.groovy](https://github.com/adoptium/temurin-build/blob/master/pipelines/build/common/build_base_file.groovy).
```

**Review by karianna — APPROVED** [review]

**Review by karianna — CHANGES_REQUESTED** [review]
Linter is complaining https://github.com/adoptium/ci-jenkins-pipelines/actions/runs/3236216890/jobs/5301716320

**Comment by lumuchris256** [comment]
closing this in favour of #468

**Pull request closed without merging**